products/compute/terraform.yaml

```diff
@@ -244,8 +244,6 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         sensitive: true
       id: !ruby/object:Overrides::Terraform::PropertyOverride
         exclude: true
-      logConfig: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
This seems unrelated?

Reply:
Whoops! Must have searched on `logConfig` and gotten mixed up!